src/server_manager/web_app/shadowbox_server.ts

```diff
@@ -302,3 +302,36 @@ export class ShadowboxServer implements server.Server {
     }
   }
 }
+
```

Comment:
shadowbox_server is really about talking to the server via the API. It's agnostic to how it was created. I'd rather keep it that way.

Perhaps create a `server_install.ts` file for the logic you are adding here.

Reply:
Done